.gitignore

```diff
@@ -95,3 +95,7 @@ build/logs
 protected/widgets/views/IndexSearch_Examination.php
 !/protected/modules/OECaseSearch/assets/css/QueryBuilder.css
 protected/tests/coverage/
+
+protected/controllers/ScratchController\.php
```

Comment:
What is the purpose of this? Why is this file unversioned?

Reply:
It's just my local testing thing, it doesn't need to end up in the repo. Scratch as in scratch pad